docs/topics/power-assert.md

```diff
@@ -0,0 +1,400 @@
+[//]: # (title: Power-assert compiler plugin)
+
+> The Power-assert compiler plugin is [Experimental](components-stability.md).
+> It may be changed at any time. Use it only for evaluation purposes.
+> We would appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
+>
+{type="warning"}
+
+The Kotlin Power-assert compiler plugin improves the debugging experience
+by providing detailed failure messages with contextual information.
+It simplifies the process of writing tests by automatically generating 
+intermediate values in failure messages. 
+It helps you understand why a test failed without needing complex assertion libraries.
```

Comment:
This paragraph mirrors the list that follows, but not quite. I'd argue for making the paragraph shorter ("improves the experience, simplifies the process...") and follow up with the list of features that confirm these claims.

Reply:
Seems valid. I'll rewrite it after.
Thanks!